docs/modules/publisher.md

```diff
@@ -143,383 +143,11 @@ There are several command line arguments that can be used to set global settings
-  >          },
-  >          "SentMessagesPerSec":"2.6",
-  >          "IngestionDuration":"{00:00:25.5491702}", 
-  >          "IngressDataChanges":"25", 
-  >          "IngressValueChanges":"103", 
-  >          "IngressBatchBlockBufferSize":"0", 
-  >          "EncodingBlockInputSize":"0",
-  >          "EncodingBlockOutputSize":"0", 
-  >          "EncoderNotificationsProcessed":"83", 
-  >          "EncoderNotificationsDropped":"0", 
-  >          "EncoderIoTMessagesProcessed":"2", 
-  >          "EncoderAvgNotificationsMessage":"41.5", 
-  >          "EncoderAvgIoTMessageBodySize":"6128", 
-  >          "EncoderAvgIoTChunkUsage":"1.158034", 
-  >          "EstimatedIoTChunksPerDay":"13526.858105160689", 
-  >          "OutgressBatchBlockBufferSize":"0", 
-  >          "OutgressInputBufferCount":"0", 
-  >          "OutgressInputBufferDropped":"0", 
-  >          "OutgressIoTMessageCount":"0", 
-  >          "ConnectionRetries":"0", 
-  >          "OpcEndpointConnected":"true", 
-  >          "MonitoredOpcNodesSucceededCount":"5", 
-  >          "MonitoredOpcNodesFailedCount":"0" 
-  >       }
-  >    ]
-  >}
-  >```
-
-#### **AddOrUpdateEndpoints_V1**
-
-This method provide the means to perform a complete `published_nodes.json` update as well as update multiple
-endpoints (DataSets) at once. Unlike `PublishNodes_V1` method, `AddOrUpdateEndpoints_V1` shall completely
-change the node set for an endpoint (DataSet) with the one provided in the method’s request payload.
-Furthermore, by providing an empty list of nodes in the request, the user can remove completely the
-previously configured nodes for a specific endpoint (DataSet).
-
-  *Request:* represents a list of objects which should strictly follow the request payload schema as
-  described above. The `OpcNodes` attribute being empty list or `null` will be interpreted as a removal
-  request for that endpoint (DataSet).
-
-  Example request payload:
-
-  ``` json
-  [
-    {
-      "EndpointUrl": "opc.tcp://opcplc:50000/",
-      "DataSetWriterGroup": "Server0",
-      "DataSetWriterId": "Device0",
-      "DataSetPublishingInterval": 5000,
-      "OpcNodes":
-      [
-        {
-          "Id": "nsu=http://microsoft.com/Opc/OpcPlc/;s=FastUInt0",
-        }
-      ]
-    },
-    {
-      "EndpointUrl": "opc.tcp://opcplc:50001/",
-      "DataSetWriterGroup": "Server1",
-      "DataSetWriterId": "Device1",
-      "OpcNodes": []
-    }
-  ]
-  ```
-
-  *Response:* when successful -  Status 200 and an empty json (`{}`) as payload
-  
-  *Exceptions:* a response corresponding to an exception will be returned if:
-
-  - request payload contains deletion request for an endpoint (DataSet) that is not present in publisher configuration.
-  - request payload contains two or more entries for the same endpoint (DataSet).
+To learn how to migrate from version 2.5 (or below) to version 2.8 for the direct methods, please check this [migration path](publisher-dmmigration.md).
```

Comment:
Let's change this to specifically mention `2.8.2` version and not `2.8` in general.